apisix/core/table.lua

```diff
@@ -61,9 +61,7 @@ end
 
 
 function _M.try_read_attr(tab, ...)
-    local count = select('#', ...)
```

Comment:
Changes below are quite trivial. We don't accept such change which breaks the `git blame`.
Please remove them.